homeassistant/components/switch/rachio.py

```diff
@@ -8,7 +8,7 @@
 from homeassistant.components.switch import SwitchDevice, PLATFORM_SCHEMA
 from homeassistant.const import CONF_ACCESS_TOKEN
 
-REQUIREMENTS = ['rachiopy==0.1.2']
+REQUIREMENTS = ['rachiopy==0.1.1']
```

Comment:
Seems like we have always used the upstream version, never your fork? We should not downgrade unless necessary. Please revert this change